tests/functional/testContactUs.js

```diff
@@ -6,69 +6,308 @@ const vpn = require('./helper.js');
+      'automated subject'
+    );
+
+    await vpn.waitForQueryAndWriteInTextField(
+      queries.screenGetHelp.contactSupportView.DESCRIBE_TEXTAREA.visible(),
+      'automated describe area'
+    );
+
+    await vpn.waitForQueryAndClick(queries.screenGetHelp.contactSupportView.SUBMIT_BUTTON.visible());
+  });
+});
+
+describe('Contact us view - authenticated user', function () {
+  this.ctx.authenticationNeeded = true;
+
+  it('VPNUserInfo visible to authenticated users', async () => {
+    await openContactUsInSettings(true);
+
+    await vpn.waitForQuery(
+      queries.screenGetHelp.contactSupportView.USER_INFO.visible()
+    );
+  });
+
+  it('VPNUserInfo is disabled', async () => {
+    await openContactUsInSettings(true);
+
+    await vpn.waitForQuery(
+      queries.screenGetHelp.contactSupportView.USER_INFO.visible().disabled()
+    );
+  });
 
-    it('Email inputs are not visible to authenticated user', async () => {
-      it('Email inputs are visible to unauthenticated user', async () => {
-        await openContactUsInSettings();
+  it('authenticated User is able to fill out form', async () => {
+    await openContactUsInSettings(true);
 
-        await vpn.waitForQuery(queries.screenGetHelp.contactSupportView
-                                   .UNAUTH_USER_INPUTS.hidden());
-      });
+    await vpn.waitForQueryAndClick(
+      queries.screenGetHelp.contactSupportView.CATEGORY_DROPDOWN.visible()
+    );
+
+    // wait for element to be interactable
+    await vpn.wait();
+
+    await Promise.all([
+      assert.equal(
+        await getDropDownTextString(0),
+        'Payment and billing'
+      ),
+      assert.equal(
+        await getDropDownTextString(1),
+        'Account issues'
+      ),
+      assert.equal(
+        await getDropDownTextString(2),
+        'Technical issues'
+      ),
+      assert.equal(
+        await getDropDownTextString(3),
+        'Request features'
+      ),
+      assert.equal(
+        await getDropDownTextString(4),
+        'Other'
+      ),
+    ]);
+
+
+    await vpn.waitForQueryAndClick('//dropdown-Option-4-text');
+    await vpn.waitForQueryAndWriteInTextField(
+      queries.screenGetHelp.contactSupportView.SUBJECT_INPUT.visible(),
+      'automated subject'
+    );
+    await vpn.waitForQueryAndWriteInTextField(
+      queries.screenGetHelp.contactSupportView.DESCRIBE_TEXTAREA.visible(),
+      'automated describe area'
+    );
+    await vpn.waitForQueryAndClick(
+      queries.screenGetHelp.contactSupportView.SUBMIT_BUTTON.visible()
+    );
+  });
+
+  it('Email inputs are not visible to authenticated user', async () => {
+    it('Email inputs are visible to unauthenticated user', async () => {
+      await openContactUsInSettings(true);
+
+      await vpn.waitForQuery(
+        queries.screenGetHelp.contactSupportView.UNAUTH_USER_INPUT.hidden()
+      );
     });
   });
 });
+
+describe('Authenticated failures', function () {
```

Comment:
We have decided to remove the authenticated support form path (https://mozilla-hub.atlassian.net/browse/VPN-3747) so we may be able to get rid of these tests. But we do still want to test that support forms work for both authenticated and unauthenticated users.